### PR TITLE
Make openstack-operator only deploy CeilometerCentral

### PR DESCRIPTION
**Pull request by jlarriba**

### CODE REVIEW

File: config/samples/core_v1beta1_openstackcontrolplane.yaml [hunk 1]
```diff
@@ -116,11 +116,10 @@ spec:
...
-  telemetry:
+  ceilometer:
     template:
-      ceilometerCentral:
-        passwordSelector:
-          service: CeilometerPassword
-        secret: osp-secret
-        serviceUser: ceilometer
-      rabbitMqClusterName: rabbitmq
+      serviceAccount: telemetry-operator-telemetry
```
Comment (abays) [inline config/samples/core_v1beta1_openstackcontrolplane.yaml:121]: This should be created by the Telemetry operator itself, so I think we should remove it

**Comment by abays** [comment]
@jlarriba I think this is almost ready.  I tested it in my env and it worked, but I used the latest Telemetry operator API.  I think as a final update to the PR, we should `go get -u github.com/openstack-k8s-operators/telemetry-operator/api` to bump it to the latest version.

**Comment by abays** [comment]
> Build failed (check pipeline). For information on how to proceed, see http://docs.openstack.org/infra/manual/developers.html#automated-testing
> 
> https://review.rdoproject.org/zuul/buildset/43d24b0eaa514a1391539de30e840f7e
> 
> heavy_check_mark [openstack-operator-content-provider ](https://review.rdoproject.org/zuul/build/d1ea2a90dfee44589d2b4c688471dc87) SUCCESS in 55m 53s x [openstack-operator-crc-podified-edpm-baremetal ](https://review.rdoproject.org/zuul/build/dddb8c12e6924315807ab21a2f107d67) **FAILURE** in 24m 01s x [openstack-operator-crc-podified-edpm-deployment ](https://review.rdoproject.org/zuul/build/c34f46da9ef64eeebc1d82d28d779e7d) **FAILURE** in 38m 32s

This failure seems unrelated to this PR:

```
Error from server (NotFound): network-attachment-definitions.k8s.cni.cncf.io \"baremetal-net\" not found
```

The other one _could_ be related, but I don't see how the changes would cause this:

```
Internal error occurred: failed calling webhook \\\"mopenstackcontrolplane.kb.io\\\": failed to call webhook: Post \\\"https://openstack-operator-controller-manager-service.openstack-operators.svc:443/mutate-core-openstack-org-v1beta1-openstackcontrolplane?timeout=10s\\\": dial tcp 10.217.0.103:9443: connect: connection refused
```
```

**Comment by jlarriba** [comment]
recheck

**Comment by jlarriba** [comment]
I rechecked, but it seems to be failing in all the PRs currently active

**Comment by abays** [comment]
> I rechecked, but it seems to be failing in all the PRs currently active

I checked with the CI team and there is a known issue preventing the Zuul jobs from working.  Since all other tests passed, I think we can merge this.

**Pull request merged**